index.d.ts

```diff
@@ -0,0 +1,15 @@
+import { Plugin } from 'fastify';
+import { Server, IncomingMessage, ServerResponse } from 'http';
+
+interface FastifyEtagOptions {
+  algorithm?: 'fnv1a' | string;
```

Comment:
I'm not sure if adding `'fnv1a'` has any useful effect here but I added it just to be documentative

Reply:
Awesome, thanks!